docs/sources/k6-studio/record-your-first-script.md

```diff
@@ -108,9 +113,11 @@ On the top, you can inspect the recording from this view, similar to the Recorde
 
 You can also inspect the script that would be generated by selecting the **Script** tab in the top panel.
 
+## TODO: Step to validate the script and use checks to see that is not working
```

Comment:
This is a placeholder to discuss moving the validation step here so we can already showcase the full workflow, starting from the failing state of the script generated straight from the recording without rules

Reply:
I moved the "Validate a test script" section up to this line, and made some changes to it. :)